src/styles/main.css

```diff
@@ -0,0 +1,301 @@
+}
+
+.day:nth-child(11)::before {
+  content: '11';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(12)::before {
+  content: '12';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(13)::before {
+  content: '13';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(14)::before {
+  content: '14';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(15)::before {
+  content: '15';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(16)::before {
+  content: '16';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(17)::before {
+  content: '17';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(18)::before {
+  content: '18';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(19)::before {
+  content: '19';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(20)::before {
+  content: '20';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(21)::before {
+  content: '21';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(22)::before {
+  content: '22';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(23)::before {
+  content: '23';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(24)::before {
+  content: '24';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(25)::before {
+  content: '25';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(26)::before {
+  content: '26';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(27)::before {
+  content: '27';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(28)::before {
+  content: '28';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(29)::before {
+  content: '29';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(30)::before {
+  content: '30';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.day:nth-child(31)::before {
+  content: '31';
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
```

Comment:
Consider using a CSS preprocessor like SCSS to generate the repetitive nth-child rules for day numbers. This can help reduce redundancy and make the code easier to maintain.